Libraries/Components/AppleTV/TVViewPropTypes.js

```diff
@@ -34,51 +39,36 @@ const TVViewPropTypes = {
    * shiftDistanceY: Defaults to 2.0.
    * tiltAngle: Defaults to 0.05.
    * magnification: Defaults to 1.0.
-   * pressMagnification: Defaults to 1.0.
```

Comment:
Do we still want these lines?

Reply:
I've never used this, do they bring useful information ?

Reply:
I'm not sure either, but it seems like we should be keeping them unless the options are no longer valid.